### PR TITLE
v2.0.3 - support for dedicated hosts and scale storage

### PR DESCRIPTION
**Pull request by alexarvinraj**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # **CHANGELOG**
 
+## **2.0.3**
+### ENHANCEMENTS
+- Support dedicated hosts for static worker nodes.
+- Support for Spectrum Scale storage nodes.
+
+### **BUG FIXES**
+- Fixed bug related to Http data source body depreciation.
```
Comment (AugieMena3) [inline CHANGELOG.md]: minor typo here... should be "deprecation"

File: scripts/LSF_dynamic_workers.sh [hunk 2]
```diff
@@ -32,7 +32,7 @@ hostnamectl set-hostname ${lsfWorkerhostname}
 host_prefix=$(hostname|cut -f1-4 -d -)
 
 # NOTE: On ibm gen2, the default DNS server do not have reverse hostname/IP resolution.
-# 1) put the master server hostname and ip into lsf hosts.
+# 1) put the controller server hostname and ip into lsf hosts.
```
Comment (AugieMena3) [inline scripts/LSF_dynamic_workers.sh:35]: We did settle on changing "controller" to "management node" in the next release, if time allows, so this is fine.  In fact, looking at all the uses of "controller" we may want to rethink that further change!  It doesn't match up with what's used externally, but we may be able to live with it within the code.

**Review by AugieMena3 — APPROVED** [review]
Only one minor typo...  "depreciation" should be "deprecation".  I'm ok leaving it there until the next release if it's a hassle to change at this point.

File: scripts/LSF_dynamic_workers.sh (hunk 2 above)
Comment (alexarvinraj) [reply scripts/LSF_dynamic_workers.sh:35]: @AugieMena3 We decided that this will be fixed in next release

File: CHANGELOG.md (hunk 1 above)
Comment (alexarvinraj) [reply CHANGELOG.md]: Fixed now

**Pull request merged**